### PR TITLE
Refactor functional area overview

### PR DESCRIPTION
**Pull request by choldgraf**
This is a medium-sized refactor of our top-level documentation with the following goals:

- Standardize the structure of several pages across areas of our organization
- Make it clearer what is organization-wide vs. area-specific
- Reduce duplicated content and follow a single source of truth

Here are the major changes:

- Each area now roughly has the four main top level sections: "Scope and responsibilities", "Structure and roles", "Workflow", and optionally "Strategy".
- I pulled out some content that used to be in a "job description" and instead put it into the area-wide docs (e.g. stuff about responsibilities and scope)
- Each major area now has a `roles/` directory and we have one page per role, rather than listing them all one one page.
- There is now an `operations/governance.md` page to define our basic decision-making principles, but we'll need to flesh this out in the future (ref: https://github.com/2i2c-org/meta/issues/489)
- There is now an "Area Leads" functional area that describes our processes for cross-organization coordination and discussion.
	- I've included a section about "scope and responsibilities" that I'd like agreement from @colliand @jmunroe @damianavila and @yuvipanda on.
- A few sub-pages have been merged, rearranged, etc to try and make things easier to follow

[**There's an overview of this structure on the main page here now**](https://2i2c-team-compass--658.org.readthedocs.build/en/658/#structure-of-the-team-compass)

### Thoughts on a review

**Reasonable structure?**. I know that this kind-of a messy and big PR. Many of the added sections are fairly small, and the goal is just to standardize the top-level sections so that people can quickly orient themselves throughout the document. Hopefully most of that is just "reorganizing" and not "adding new things". Is this a reasonable structure to use?

**Area leads content?**. The biggest update is the extra content in the "leads" section. I'd love thoughts on whether the [Area Leads](https://2i2c-team-compass--658.org.readthedocs.build/en/658/leads/index.html) section is a helpful way of describing our cross-area coordination efforts.

### To do

- [ ] Give it another read and decide if there's anything missing
- [ ] Address any feedback from team
- [ ] See if we have the right leadership information there to be able to move forward while I'm on paternity leave https://github.com/2i2c-org/meta/issues/487

### References

- closes https://github.com/2i2c-org/meta/issues/486
- closes https://github.com/2i2c-org/meta/issues/488
- closes https://github.com/2i2c-org/meta/issues/487

### CODE REVIEW

**Comment by colliand** [comment]
Thanks @choldgraf. My quick reaction is very positive. It will take me some time to review in more detail. Thanks for putting in this work!

File: leads/governance.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+This section describes more specific ways in which decisions usually happen.
+
+## Decision-making scope
+
+Generally speaking, we try to push decision-making as close to where the action happens as possible.
+This means that most decisions are made by individuals without requiring a consultation with specific leaders in the organization.
+
+### Within-area decisions
+
+For decisions that primarily affect their area and no others, team leads are empowered to make decisions on their own without requiring consent from other team members.
```
Comment (choldgraf) [inline leads/governance.md:13]:
```suggestion
For decisions that primarily affect their area and no others, team leads are empowered to make decisions within their own team without requiring consent from team members of other areas.
```

File: leads/overview.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Scope and responsibilities
+
+The Team Leads is a bit different than most other 2i2c teams.
```
Comment (choldgraf) [inline leads/overview.md:3]:
```suggestion
The Team Leads group is a bit different than most other 2i2c teams.
```

File: leads/structure.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Structure and roles
+
+The team leads is composed of leaders from various areas of 2i2c.
```
Comment (choldgraf) [inline leads/structure.md:3]:
```suggestion
The team leads group is composed of leaders from various areas of 2i2c.
```

File: leads/workflow.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+(leads:workflow)=
+# Workflow
+
+This section describes how our leadership team carries out its planning and day-to-day work.
+
+(leads:meeting:organizational-strategy)=
+## Meeting for organizational strategy
+
+We have a bi-weekly **Organization and strategy meeting** for to discuss and refine major priorities across the organization.
```
Comment (choldgraf) [inline leads/workflow.md:9]:
```suggestion
We have a bi-weekly **Organization and strategy meeting** to discuss and refine major priorities across the organization.
```

File: leads/workflow.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+We have a special project board to track the priorities of each functional area.
+We update this roughly every quarter.
+
+```{button-link} https://github.com/orgs/2i2c-org/projects/34/views/7
+:color: primary
+
+Strategic priorities
+```
+
+## Backlog board
```
Comment (choldgraf) [inline leads/workflow.md:23]:
```suggestion
## Organization-wide backlog
```

File: organization/structure.md [hunk 6]
```diff
@@ -16,16 +16,60 @@ We nonetheless document these structures to understand how our work and responsi
...
+- **[Administration](../administration/index.md)**: Carry out ongoing organizational tasks that are needed to keep 2i2c running on a daily basis.
+
+## Functional areas
+
+Functional areas are organized around _responsibilities, skills, and career tracks_.
+They have dedicated leads that are defined in the **Structure and roles** section of each area.
+
+Reporting lines are within a functional area, meaning that team members should report to somebody that has held a similar job in the past.
+
+% TODO: Each of these major areas should havea  `<NAME>/structure.md` page that describes that organization's structure and roles (or, a similar page with this information). We can link to each from here.
```
Comment (choldgraf) [inline organization/structure.md:38]:
```suggestion
```

**Comment by damianavila** [comment]
This is a massive PR, so I took a look at the final preview and I will comment on the specific pieces outlined at the top message:

> Standardize the structure of several pages across areas of our organization

I like the standarization I saw!!

> Make it clearer what is organization-wide vs. area-specific

That distintion makes way easier to navigate the site now. In the previous iteration you had a really long list with intermixed concepts.

> Reduce duplicated content and follow a single source of truth

💯 

> Each area now roughly has the four main top level sections: "Scope and responsibilities", "Structure and roles", "Workflow", and optionally "Strategy".

Makes sense to me as important groups on each area.

> I pulled out some content that used to be in a "job description" and instead put it into the area-wide docs (e.g. stuff about responsibilities and scope)

👍 

> Each major area now has a roles/ directory and we have one page per role, rather than listing them all one one page.

Super, that will help at the time to introduce new roles on each area.

> There is now an operations/governance.md page to define our basic decision-making principles, but we'll need to flesh this out in the future (ref: https://github.com/2i2c-org/meta/issues/489)

I think it is a good start!

> There is now an "Area Leads" functional area that describes our processes for cross-organization coordination and discussion.

Super!! "Area Leads" sounds a little bit weird to me, but the name is not actually important... I think the acknoledge that we have multistakeholders and cross-sectional new areas composed by interests/needs from different existing areas is conceptually important not only for the Area Leads, particularly, but also for future cross-sectional areas (ie. Support).

> I've included a section about "scope and responsibilities" that I'd like agreement from @colliand @jmunroe @damianavila and @yuvipanda on.

The responsabilities outlined LGTM (we continuosly talk about those, thanks for writing those down).

> A few sub-pages have been merged, rearranged, etc to try and make things easier to follow

Great!

> Is this a reasonable structure to use?

Yep, it is definetely an improvement compared with the previous iteration.

> Area leads content?. The biggest update is the extra content in the "leads" section. I'd love thoughts on whether the Area Leads section is a helpful way of describing our cross-area coordination efforts.

Already provided feedback, I like it (although I do not like the name 😉 but that is totally minor a nor a blocker at all!!!).

**Review by damianavila — APPROVED** [review]
LGTM once the conflict are resolved.

**Comment by choldgraf** [comment]
> "Area Leads" sounds a little bit weird to me, but the name is not actually important... I think the acknoledge that we have multistakeholders and cross-sectional new areas composed by interests/needs from different existing areas is conceptually important not only for the Area Leads, particularly, but also for future cross-sectional areas (ie. Support)

Yeah I agree that is kind-of an awkward name. I think it's awkward because it's defined by the "kinds of people that tend to be in that team" rather than the team's focus area (which is what we use for all the others).

Maybe we should call it `Cross-team leadership`? Maybe that more clearly evokes "a team for cross-team communication and planning, as well as cross-team policy and practices"?

**Comment by damianavila** [comment]
> Maybe we should call it Cross-team leadership? Maybe that more clearly evokes "a team for cross-team communication and planning, as well as cross-team policy and practices"?

That name is better, IMHO (still awkward but works - and I do not have better suggestions from the top of my mind, so let's go with that one).

**Comment by choldgraf** [comment]
I'd like to get this one in sooner than later. Pinging @colliand and @jmunroe for a review if they have the time. If I don't hear any objections then I'll merge this tomorrow CET.

**Comment by choldgraf** [comment]
Spending some baby napping time to try and tie off some loose ends. It's been quite a long time and this PR touches a lot, so I'm going to merge it so that we can iterate from there. @damianavila you mentioned you were going to note that "support is the responsibility of engineering" by nesting it under engineering's "scope and responsibilities" section, so this should unblock you to do that.

**Comment by damianavila** [comment]
> so this should unblock you to do that.

See https://github.com/2i2c-org/team-compass/pull/680.

**Pull request merged**